app/routes/user.js

```diff
@@ -21,6 +22,46 @@ module.exports = [
       return h.response(undefined).code(204)
     }
   },
+  {
+    method: 'GET',
+    path: '/user/me/licence',
+    options: {
+      tags: ['api'],
+      notes: ['Checks if the calling user has accepted the licence'],
+      response: {
+        status: {
+          200: undefined,
+          401: undefined,
+          404: undefined
+        }
+      }
+    },
+    handler: async (request, h) => {
+      const res = await userVerifyLicenceAccepted(request)
+
+      return h.response(res).code(200)
+    }
+  },
+  {
+    method: 'PUT',
+    path: '/user/me/licence',
+    options: {
+      tags: ['api'],
+      notes: ['Checks if the calling user has accepted the licence'],
+      response: {
+        status: {
+          200: undefined,
+          401: undefined,
+          404: undefined
+        }
```

Comment:
does the sequelize query handle the 404 itself?  we probably don't need 401 here.

Reply:
Removed 401

Reply:
CHanged error condition to 500